frontend/public/components/utils/documentation.tsx

```diff
@@ -31,7 +31,7 @@ export const documentationURLs: documentationURLsType = {
   },
   configuringMonitoring: {
     downstream:
-      'html/monitoring/configuring-the-monitoring-stack#maintenance-and-support_configuring-the-monitoring-stack',
+      'html/observability/monitoring/configuring-the-monitoring-stack#maintenance-and-support_configuring-the-monitoring-stack',
     upstream:
       'monitoring/configuring-the-monitoring-stack.html#maintenance-and-support_configuring-monitoring',
```

Comment:
It appears [the upstream link](https://docs.okd.io/latest/observability/monitoring/configuring-the-monitoring-stack.html#maintenance-and-support_configuring-monitoring) has also changed.  
```suggestion
      'observability/monitoring/configuring-the-monitoring-stack.html#maintenance-and-support_configuring-monitoring',
```